src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResourceTest.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.resource
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.incentivesapi.integration.SqsIntegrationTestBase
+
+class IncentiveReviewsResourceTest : SqsIntegrationTestBase() {
+  @BeforeEach
+  internal fun setUp() {
+    offenderSearchMockServer.resetAll()
+  }
+
+  @Test
+  fun `requires a valid token to retrieve data`() {
```

Comment:
You could add additional tests for other HTTP error statuses such as `400` and `403`. See `IepLevelResourceTest.SyncIepReview` for examples

Reply:
ok, i've added a 403 test, but it's worth nothing that the majority of endpoints do not have this test coverage – it probably should be done at some point, no?

alas it'll not return a 400 because the validation is broken _throughout_. i've raised a separate ticket.

Reply:
> majority of endpoints do not have this test coverage – it probably should be done at some point, no?

Yes - we can raise a separate story in the backlog for that